junit-jupiter-api/src/main/java/org/junit/jupiter/api/condition/EnabledOnOs.java

```diff
@@ -24,10 +24,11 @@
 /**
  * {@code @EnabledOnOs} is used to signal that the annotated test class or
  * test method is only <em>enabled</em> on one or more specified
- * {@linkplain #value operating systems}.
+ * {@linkplain #value operating systems} or one or more specified
+ * {@linkplain #architectures architectures}.
  *
  * <p>When applied at the class level, all test methods within that class
- * will be enabled on the same specified operating systems.
+ * will be enabled on the same specified operating systems or architectures.
```

Comment:
We need to clearly document AND vs. OR semantics, potentially in a dedicated paragraph.